sdk/Dockerfile

```diff
@@ -0,0 +1,10 @@
+ARG BASE_IMAGE=python:3.11-slim
```

Comment:
todo, I should change the default here to be the secure one dave was talking about

Reply:
Punting on this one. Iron bank doesn't have a container registry